sway/sway-input.5.scd

```diff
@@ -214,6 +214,16 @@ correct seat.
 	disables hiding the cursor. The minimal timeout is 100 and any value less
 	than that (aside from 0), will be increased to 100.
 
+*seat* <name> keyboard_grouping none|keymap
+	Set how the keyboards in the seat are grouped together. Currently, there
+	are two options. _none_ will disable all keyboard grouping. This will make
+	it so each keyboard device has it's own isolated state. _keymap_ will
+	group the keyboards in the seat by their keymap. This is useful for when
+	the keyboard appears as multiple separate input devices. In this mode,
+	the effective layout and repeat info are also synced between the keyboards
+	in the group. The default is _keymap_. The restore the behavior of older
```

Comment:
Typo: "To Resotre the behviour of older…"

Reply:
Fixed